src/components/brew-input.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {connect} from 'react-redux';
+
+import {addBrew} from '../database/brew';
+import {FancyInput, FancyDropdown, FancyButton} from './input-stuff.js'
+
+
+var testEquip = ["Chemex", "Clever", "French Press"];
+// Eventually this would be stored elsewhere, yes?
+
+
+const FirstPanel = (props) => {
+  return (
+    <div>
+      <FancyInput label="Coffee Name " name="name" handleChange={props.handleChange}/>
+      <FancyDropdown label="Brew Method " name="brewMethod" options={testEquip}
+        handleChange={props.handleChange}/>
+      <FancyInput label="Brew Date" name="brewDate" handleChange={props.handleChange} />
+    </div>
+  )
+}
+
+const SecondPanel = (props) => {
+  return (
+    <div>
+      <FancyInput label="Brew Time " name="time" />
+      <FancyInput label="Amount (g) " name="amount" />
+      <FancyInput label="Grind Setting " name="grindSetting" />
+      <FancyInput label="Notes " name="notes" />
+      <FancyButton name="brewButton" />
+    </div>
+ )
+}
+
+const ThirdPanel = (props) => {
+  return <div> Component 3 </div>
+}
+
+
+class FancyCarouselForm extends React.Component {
+  constructor (props){
+    super(props);
+    this.state = {page:1}
+    //do I have to intialiaze form data? keep it separate?
+  }
+
+  ///handleChange = (evt) => this.setState({name: evt.target.value});
+  handleChange = (evt) => console.log(name + evt.target.value);
```

Comment:
`name` here is coming from nowhere, you probably meant `evt.target.name`

when i am trying to get a handle on these onChange events, i usually do the following:
```js
handleChange = evt => console.log(evt, evt.nativeEvent);
```